modules/update-upgrading-cli.adoc

```diff
@@ -72,7 +72,10 @@ $ oc adm upgrade
 ----
 Cluster version is 4.9.23
 
-Updates:
+Upstream: https://openshift-release.apps.ci.l2s4.p1.openshiftapps.com/graph
```

Comment:
By default, there is not spec.upstream in cv. To be consistent with the example output in step 2. I suggest to update it with default info as following:
```
Upstream is unset, so the cluster will use an appropriate default.
```